cryptocurrency-subdomain/src/main/kotlin/com/ajax/cryptocurrency/application/ports/service/CryptocurrencyService.kt

```diff
@@ -0,0 +1,16 @@
+package com.ajax.cryptocurrency.application.ports.service
+
+import com.ajax.cryptocurrency.domain.CryptocurrencyDomain
+import reactor.core.publisher.Flux
+import reactor.core.publisher.Mono
+import java.io.File
+
+interface CryptocurrencyService {
```

Comment:
lets call it CryptocurrencyInPort to highlight that this is input port for infrastructure components

Reply:
done